lib/compilation-helpers.js

```diff
@@ -121,6 +122,40 @@ function transformPath(prefix, destPath, shouldMinify) {
   };
 }
 
+/**
+ * Find the additional plugins to use in PostCSS
+ * compilation. Either accept the values from command
+ * arguments and require them in or use the config file
+ *
+ * @param {Object} argv
+ * @returns {Array}
+ */
+function determinePostCSSPlugins(argv) {
+  const config = amphoraFs.tryRequire(`${process.cwd()}/claycli.config`);
+
+  if (config && config.styles && config.styles.plugins) {
```

Comment:
I thought we wanted to make all of the cli flags root-level properties in the config?

Reply:
Are there plugins for script/template/media compilation? I didn't know if we'd end up clashing with something in the future. Easy to move up a level if we want